readme/vaccination.md

```diff
@@ -0,0 +1,31 @@
+## Vaccination
+This report searches different vaccinations. It is a MSPP Report requested by the Haitian MoH to any clinic in Haiti.
+
+**Setup**
+
+It is setup by setting the following properties in [initializer](https://github.com/mekomsolutions/openmrs-module-initializer) `jsonkeyvalues`' domain configuration. 
+
+```bash
+{
+    ...
+    ...,
+        "report.vaccination.active": "true",
+        "report.ecv.active": "true",
+        "report.prenatalVisitType": "uuid-to-prenatal-visitType",
+        "report.vaccinations": "uuid-to-vaccinations-question-concept",
+        "report.vaccinationSequenceNumber": "uuid-to-vaccination-sequence-number-concept",
+        "report.boosterSequenceNumber": "uuid-to-vaccination-booster-number-concept",
+        "report.vaccinationList":"uuids-to-vaccinations:booster-or-sequence-number",
+        "report.ecvList" : "uuid-to-ecv:booster-or-sequence-number"
+}
+```
+`report.vaccination.active` activates the individual vaccinations report to be usable when the module is loaded.
+`report.vaccination.active` activates the ECV part of report to be usable when the module is loaded.
+`report.prenatalVisitType` specifies the uuid  of prenatal visit to filter on.
+`report.vaccinations` specifies the uuid of vaccination question concept.
+`report.vaccinationSequenceNumber`specifies uuid of the vaccination sequence number concept.
+`report.boosterSequenceNumber` specifies the uuid of the vaccination booster number concept.
+`report.vaccinationList` specifies the uuids of individual vaccinations with their sequence or booster number separated by a full-colon.
```

Comment:
"UUIDs"
Could you provide just a minimal example here?

Reply:
Done